tests/clone_test.go

```diff
@@ -869,3 +849,23 @@ func stopCloneVM(virtClient kubecli.KubevirtClient, vm *virtv1.VirtualMachine) (
 
 	return virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, v1.PatchOptions{})
 }
+
+func withFirmware(firmware *virtv1.Firmware) libvmi.Option {
+	return func(vmi *virtv1.VirtualMachineInstance) {
+		vmi.Spec.Domain.Firmware = firmware
+	}
+}
+
+func createSourceVM(options ...libvmi.Option) *virtv1.VirtualMachine {
+	vmi := libvmifact.NewCirros(options...)
+	vmi.Namespace = testsuite.GetTestNamespace(nil)
+	vm := libvmi.NewVirtualMachine(vmi,
+		libvmi.WithAnnotations(vmi.Annotations),
+		libvmi.WithLabels(vmi.Labels))
+
+	By(fmt.Sprintf("Creating VM %s", vm.Name))
+	virtClient := kubevirt.Client()
+	updatedVm, err := virtClient.VirtualMachine(vm.Namespace).Create(context.Background(), vm, v1.CreateOptions{})
+	Expect(err).ShouldNot(HaveOccurred())
```

Comment:
nit: Please consider returning and error and assert at the caller, or user asjust the assert offset (ExpectWithOffset).

Reply:
ahh you're totally right!